packages/types/index.d.ts

```diff
@@ -258,4 +258,9 @@ declare module '@walletconnect/types' {
     rpc?: IRPCMap
     infuraId?: string
   }
+
+  export interface IRequestOptions {
+    _topic?: string
```

Comment:
Remove custom topic option

Reply:
Moved topic back to separate param, and made options second optional param